resources/sql/backtick.sql

```diff
@@ -20,9 +20,9 @@ returning bq.*;
 -- name: queue-insert<!
 -- Insert a new job element
 insert into backtick_queue
-  (name, priority, state, tries, data, run_at, started_at, created_at, updated_at)
+  (name, priority, state, tries, data, started_at, created_at, updated_at)
 values
-  (:name, :priority, :state, :tries, :data, :run_at, now(), now(), now());
+  (:name, coalesce(:priority, now()), :state, :tries, :data, now(), now(), now());
```

Comment:
I thought that was a mistake also, but didn't want to add too much to the current diff.

I'll do another diff with that patch.